### DISCUSSION TITLE
Follow me on github I promise id follow you back

### DISCUSSION THREAD

**Discussion by Blockchain-Oracle** [Cyfrin/foundry-full-course-cu#2232] [General]
[Github link ](https://github.com/Blockchain-Oracle)

**Comment by EngrPips** [comment]
Interesting.

**Comment by jovylle** [comment]
Hey Hi there, follow me back too.

**Reply by jovylle** [reply]
@Blockchain-Oracle i'm waiting for your follow back, reply here if you active so I can follow you

**Reply by Blockchain-Oracle** [reply]
ok chim followed you back already

**Reply by EngrPips** [reply]
nice

**Comment by EngrPips** [comment]
Interesting.
